working-open.md

```diff
@@ -51,3 +53,10 @@ Hypha operates with democratic organizing principles informed by our [values](./
 - Chat transcripts
 - Notes involving external parties and collaborators
 - Materials from project retrospectives
+
+**Sensitive and confidential**:
+
+- Passwords, credentials, and sensitive personal information
+- Data sets, software, and other content in project work that we don't have consent to share
+- Financial or other information of our clients and partners that we don't have consent to share
+- Data collected through our digital infrastructure, which may contain user account information or personally identifiable information (PII)
```

Comment:
```suggestion
- Data collected through our digital infrastructure, which may contain user account information or personal information
```
EDIT: suggest just using personal information to be consistent, and use the PIPEDA definition I put above, which doesn't have PII

Reply:
I think this is fine, and we do not need to link to PIPEDA or expand on it.